chalice/deploy/sweeper.py

```diff
@@ -78,79 +170,8 @@ def _plan_deletion(self,
         # type: (...) -> None
         for name in remaining:
             resource_values = deployed.resource_values(name)
-            if resource_values['resource_type'] == 'lambda_function':
```

Comment:
Fwiw, This for/if loop refactoring was triggered by hitting flake8  check on cyclomatic complexity limits when adding another conditional.